_includes/tutorials/kafka-producer-application/kafka/code/configuration/prod.properties

```diff
@@ -0,0 +1,8 @@
+bootstrap.servers=<FILL ME IN>
+
+key.serializer=org.apache.kafka.common.serialization.StringSerializer
+value.serializer=org.apache.kafka.common.serialization.StringSerializer
+acks=all
+
+
```

Comment:
Is input topic required here, like the other stages?

Reply:
ack